src/MockBot.Api/Services/ChatService.cs

```diff
@@ -0,0 +1,39 @@
+﻿using MockBot.Api.Interfaces;
+using MockBot.Api.Models;
+
+namespace MockBot.Api.Services
+{
+    public class ChatService : IChatService
+    {
+        private readonly Dictionary<Guid, Chat> _chats;
```

Comment:
I would recommend making this an `IDictionary`
```suggestion
        private readonly IDictionary<Guid, Chat> _chats;
```

and new it up as a `ConcurrentDictionary`.

Since we plan to load test DMR, there could potentially be a lot of concurrent requests hitting MockBot as a result and we don't want concurrency or other indexing issues causing exceptions.